mmv1/products/networkservices/TlsRoute.yaml

```diff
@@ -0,0 +1,159 @@
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tls_route_mesh_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tls-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tls_route_gateway_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tls-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    url_param_only: true
+    immutable: true
+    description: |
+      Name of the TlsRoute resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'selfLink'
+    description: |
+      Server-defined URL of this resource.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the TlsRoute was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the TlsRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::Array
+    name: meshes
+    item_type: Api::Type::String
+    description: |
+          List of meshes this TlsRoute is attached to, as one of the routing rules to route the requests served by the mesh.  
+    send_empty_value: true 
+  - !ruby/object:Api::Type::Array
+    name: gateways
+    item_type: Api::Type::String
+    description: |
+          List of gateways this TlsRoute is attached to, as one of the routing rules to route the requests served by the gateway.  
+    send_empty_value: true
+  - !ruby/object:Api::Type::Array
+    name: rules
+    description: |
+          Rules that define how traffic is routed and handled.  
+    required: true
+    send_empty_value: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties: 
+        - !ruby/object:Api::Type::Array
+          name: matches
+          description: |
+            Matches define the predicate used to match requests to a given action.
+          required: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::Array
+                name: sniHost
+                description: |
+                  SNI (server name indicator) to match against.
+                item_type: Api::Type::String
+                send_empty_value: true
+              - !ruby/object:Api::Type::Array
+                name: alpn
+                description: |
+                  ALPN (Application-Layer Protocol Negotiation) to match against.
```

Comment:
It would also be good to update these to include the "At least one of sniHost and alpn is required." and "Up to 5" details

Reply:
Added